erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -116,6 +135,104 @@ void BandwidthEstimationHandler::read(Context *ctx, std::shared_ptr<dataPacket>
       ELOG_DEBUG("Packet not parsed %d", packet->type);
     }
   }
+
+  uint32_t now = ClockUtils::timePointToMs(clock::now());
+  RtpHeader *head = reinterpret_cast<RtpHeader*> (packet->data);
+  if (!chead->isRtcp()) {
+    // RTP PACKETS
+    uint16_t seqNum = head->getSeqNumber();
+    switch (packet->type) {
+      case VIDEO_PACKET:
+      // RTP VIDEO PACKET
+        videoRR.ssrc = head->getSSRC();
+        if (!rtpSequenceLessThan(seqNum, videoRR.last_rtp_seq_num)) {
+          if (seqNum < videoRR.last_rtp_seq_num) {
+            ELOG_WARN("VIDEO RTP WRAPPED");
+            videoRR.cycle++;
+          }
+          videoRR.last_rtp_seq_num = seqNum;
+        }
+      break;
+      case AUDIO_PACKET:
+      // RTP AUDIO PACKET
+        audioRR.ssrc = head->getSSRC();
+        if (!rtpSequenceLessThan(seqNum, audioRR.last_rtp_seq_num)) {
+          if (seqNum < audioRR.last_rtp_seq_num) {
+            ELOG_WARN("VIDEO RTP WRAPPED");
+            audioRR.cycle++;
+          }
+          audioRR.last_rtp_seq_num = seqNum;
+        }
+        break;
+        default:
+        break;
+    }
+  } else {
+    // RTCP PACKETS
+    if (chead->getSSRC() == videoRR.ssrc && chead->packettype == RTCP_Sender_PT) {
+      // VIDEO SR
+      videoRR.last_sr_mid_ntp = chead->get32MiddleNtp();
+      videoRR.last_sr_recv_ts = packet->received_time_ms;
+    }
+    if (chead->getSSRC() == audioRR.ssrc && chead->packettype == RTCP_Sender_PT) {
+      // AUDIO SR
+      audioRR.last_sr_mid_ntp = chead->get32MiddleNtp();
+      audioRR.last_sr_recv_ts = packet->received_time_ms;
+    }
+  }
+
+  if (videoRR.ssrc != 0 && (now - videoRR.last_rr_sent_ts > kRRSendIntervalMs)) {
+    // SEND A VIDEO RR
+    uint32_t highestSeq = videoRR.cycle;
+    highestSeq = (highestSeq << 16) | videoRR.last_rtp_seq_num;
+    RtcpHeader rtcpHead;
+    rtcpHead.setPacketType(RTCP_Receiver_PT);
+    rtcpHead.setSSRC(videoRR.ssrc);
+    rtcpHead.setSourceSSRC(videoRR.ssrc);
+    rtcpHead.setFractionLost(0*256);
```

Comment:
I'm a bit worried about not sending actual measurements here. Congestion Control mechanism will take wrong actions if we don't send packet losses and jitter, right?